local_planner/include/local_planner/local_footstep_planner.h

```diff
@@ -423,14 +428,17 @@ class LocalFootstepPlanner {
   /// Weighting on the projection of the grf
   double grf_weight_;
 
-  /// Primitive ids - FLIGHT
-  const int FLIGHT = 0;
+  /// Primitive ids - CONNECT_STANCE
+  const int CONNECT_STANCE = 0;
```

Comment:
Maybe in the next refactoring pr, these can be enums instead.

Reply:
Agreed, that's on my todo list. Honestly there are a few instance of these that we should handle more uniformly.

Reply:
Yeah maybe add a comment here so we remember?